Mathlib/Topology/Algebra/InfiniteSum/Basic.lean

```diff
@@ -215,6 +215,10 @@ theorem hasSum_single {f : β → α} (b : β) (hf : ∀ (b') (_ : b' ≠ b), f
   hasSum_sum_of_ne_finset_zero <| by simpa [hf]
 #align has_sum_single hasSum_single
 
+lemma hasSum_singleton (m : β) (f : β → α) : HasSum (fun x : ({m} : Set β) ↦ f x) (f m) := by
+  convert_to HasSum (fun x : ({m} : Set β) ↦ f x) (f (⟨m, rfl⟩ : ({m} : Set β)))
+  exact hasSum_single (α := α) _ <| fun m' h ↦ False.elim <| h <| Subtype.ext m'.2
+
```

Comment:
In the statement, one can replace `(fun x : ({m} : Set β) ↦ f x)` with `(({m} : Set β).restrict f)` which is slightly shorter and IMHO also more readable. Would that change break other code of yours downstream of this?

Reply:
OK. It's defeq, and nothing breaks.